tests/src/com/unciv/logic/event/EventBusTest.kt

```diff
@@ -20,6 +21,17 @@ class EventBusTest {
         assertThat(callCount, `is`(1))
     }
 
+    @Test
+    fun `should receive parent event when child event with two levels of inheritance is sent`() {
+        val events = EventBus.EventReceiver()
+        var callCount = 0
+        events.receive(Parent2::class) { ++callCount }
+
+        EventBus.send(Child())
+
+        assertThat(callCount, `is`(1))
```

Comment:
What's wrong with the Assert.assertTrue(callCount == 1) used in other tests? Is that hamcrest core a new library?